policyengine_us/variables/gov/states/hi/tax/income/credits/cdcc/dependent_care_benefit/hi_cdcc_min_head_spouse_earned.py

```diff
@@ -0,0 +1,50 @@
+from policyengine_us.model_api import *
+
+
+class hi_cdcc_min_head_spouse_earned(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Hawaii minimum income between head and spouse for the CDCC"
+    defined_for = StateCode.HI
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=28"
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=29"
+        "https://files.hawaii.gov/tax/legal/hrs/hrs_235.pdf#page=41"
+        "https://files.hawaii.gov/tax/forms/2022/schx_i.pdf#page=2"
+    )
+
+    def formula(tax_unit, period, parameters):
+        # Head or spouse are eligible for an income floor if disabled or a student
+        # Edge case: both spouses were students or disabled:
```

Comment:
They should floor the smaller, not larger, earnings.
```suggestion
        # If both spouses are students or disabled, only one can claim the floor.
        # To maximize the credit value, since the credit phases in with respect to (floored) earnings,
        # the filer should floor the spouse with the smaller earnings.
        # Note that the phase-out is separate and with respect to AGI.
```